### PR TITLE
Add pin assignment view stab files

### PR DESCRIPTION
**Pull request by KochynVolodymyr**
Add Pin Assignment View stab files

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -100,6 +100,7 @@ add_subdirectory(src/IPGenerate)
 add_subdirectory(src/TextEditor)
 add_subdirectory(src/Utils)
 add_subdirectory(src/IpConfigurator)
+add_subdirectory(src/PinAssignmentView)
```
Comment (ravic-rs) [inline CMakeLists.txt:103]: Change to PinAssignment to align with other srcs.

File: Makefile [hunk 2]
```diff
@@ -157,6 +157,7 @@ test/gui: run-cmake-debug
 	$(XVFB) ./dbuild/bin/foedag --replay tests/TestGui/gui_task_dlg.tcl
 	$(XVFB) ./dbuild/bin/foedag --replay tests/TestGui/gui_top_settings_dlg.tcl
 	$(XVFB) ./dbuild/bin/foedag --replay tests/TestGui/gui_ipconfigurator.tcl
+	$(XVFB) ./dbuild/bin/pinassignment --replay tests/TestGui/gui_pinassignment.tcl
```
Comment (ravic-rs) [inline Makefile:160]: Add to test/valgrind

**Review by ravic-rs — CHANGES_REQUESTED** [review]
Few comments to address.

File: CMakeLists.txt (hunk 1 above)
Comment (KochynVolodymyr) [reply CMakeLists.txt]: Done

File: Makefile (hunk 2 above)
Comment (KochynVolodymyr) [reply Makefile:162]: Done

**Review by ravic-rs — APPROVED** [review]
Looks good.

**Pull request merged**